frontend/packages/console-app/src/components/console-operator/ConsolePluginCSPStatusDetail.tsx

```diff
@@ -9,7 +9,7 @@ const ConsolePluginCSPStatusDetail: React.FC<DetailsItemComponentProps> = ({ obj
   const pluginStore = usePluginStore();
   const pluginName = React.useMemo(() => obj?.metadata?.name, [obj?.metadata?.name]);
 
-  const pluginInfo = React.useMemo(() => pluginStore.findDynamicPluginInfo(pluginName), [
+  const pluginInfo = React.useMemo(() => pluginStore.findDynamicPluginInfo(pluginName ?? ''), [
```

Comment:
It would be better to be consistent with the default value for `pluginName = obj?.metadata?.name`. The 'unknown'  is used as the default in another place.